app/src/jembi/res/xml/preferences.xml

```diff
@@ -4,7 +4,7 @@
             <intent
                 android:action="android.intent.action.MAIN"
                 android:targetPackage="org.fdroid.fdroid"
-                android:targetClass="org.fdroid.fdroid.AboutActivity" />
+                android:targetClass="AboutActivity" />
```

Comment:
Not sure about this change on its own - how would it find the AboutActivity without a package name?